code/src/App.js

```diff
@@ -1,29 +1,34 @@
-import React from "react";
-import { BarcodeScanner } from "components/BarcodeScanner";
+import React from 'react';
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+import { Provider } from 'react-redux';
+import { productStore } from 'reducers/productStore';
+import { ScanBarcode } from './components/ScanBarcode';
+import { Result } from 'components/Result';
+import { Header } from './components/Header';
+import styled from 'styled-components';
 
-const onDetected = (code) => {
-  console.log(`Code: ${code}`);
-  fetch(`https://world.openfoodfacts.org/api/v0/product/${code}.json`)
-    .then((data) => data.json())
-    .then((json) => {
-      console.log(json);
-    });
-};
+const ParentContainer = styled.div`
+	width: 100%;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: space-evenly;
```

Comment:
Check your indentation settings in vs code. Looks like you're using tabs instead of spaces. You should set vs code to use 2 spaces for indentation.